src/content/contributing/adding-layer-2/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Adding Layer 2's
+description: The policy we use when adding layer 2's to ethereum.org
+lang: en
+sidebar: true
+---
+
+# Adding layer 2's {#adding-layer-2}
+
+Anyone is free to suggest new layer 2's on ethereum.org.
+
+We currently list them on:
+
+- [ethereum.org/layer-2](/layer-2/)
+
+This page is a beginner/intermediate level page on layer 2's, and on this page a user is able to see some prominent layer 2 projects that they can easily get onboarded on.
+
+Because of this context, we need some specific information when you suggest a layer 2.
+
+## What we need {#what-we-need}
+
+Some information we will need beyond standard information of the layer 2 are:
```

Comment:
"standard information of the layer 2" is pretty vague, what are you thinking here? Perhaps we say in addition to the information we request from a [product](https://ethereum.org/en/contributing/adding-products/) or [developer tool](https://ethereum.org/en/contributing/adding-developer-tools/)?

Reply:
To avoid confusion let's be explicit on this page **exactly** what information we require instead of expecting users to combine this page with some information from the product/developer tools.

Reply:
Agreed 💯